packages/rocketchat-custom-sounds/client/admin/soundInfo.js

```diff
@@ -90,15 +90,15 @@ Template.soundInfo.onCreated(function() {
 
 	this.autorun(() => {
 		const data = Template.currentData();
-		if (data.clear != null) {
+		if (data != null && data.clear != null) {
 			this.clear = data.clear;
 		}
 	});
 
 	this.autorun(() => {
 		const data = Template.currentData();
 		const sound = this.sound.get();
-		if (sound.name != null) {
+		if (sound != null && sound.name != null) {
```

Comment:
Same here

Reply:
OK.
By the way, i just let it the same with the `emojiInfo.js`